src/App.jsx

```diff
@@ -0,0 +1,31 @@
+import './App.css'
+
+function App() {
+
+  const genLinks = () =>{
+    let arr = []
+    for(let i = 0; i< 10; i++){
+      arr.push({text : `Exercise ${i}`, link: `/exercise${i}`})
+    }
+    return arr;
+  }
+
+  const array = genLinks()
+  const array1 = ['dog', 'cat', 'chicken', 'cow', 'sheep', 'horse']
+  return (
+    <div className="h-screen flex flex-col items-center ">
+      <div>
+        {array.map((item, index)=>(
+          <a key={index} href={item.link} className={`px-2 underline underline-offset-4 ${!index == 0 && "border-indigo-600 border-l"} text-indigo-600 `} >{index == 0 ? "Home" : item.text}</a>
+        ))}
+      </div>
+      <ul className='mt-32 flex flex-col items-start'>
+          {array1.map(item=>(
+            <li key={item} className="list-disc">{item}</li>
```

Comment:
It is fine to assign the value of the **key** to the **item** as they are different in your array, but it is generally better to pick values with zero chances of ever being the same such as the indices or Ids if they are available. This is because in some arrays items might have the same values, but indices or Ids are never the same. So my suggestion would be to assign your key as this,

```suggestion
          {array1.map((item,index)=>(
            <li key={index} className="list-disc">{item}</li>
```